lib/signatures.js

```diff
@@ -81,8 +81,8 @@ function validate( entries ) {
 
 		// Map to name and email
 		return {
-			names: [ unorm.nfc( row.gsx$fullname.$t ) ],
-			email: row.gsx$emailaddress.$t,
+			names: [ unorm.nfc( row.gsx$fullname.$t.toLowerCase() ) ],
```

Comment:
I'll drop this `toLowerCase()` and modify the `noMatchingNames` check to be case-insensitive.